applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -272,9 +271,10 @@ int bt_le_audio_tx_send(struct bt_bap_stream **bap_streams, struct le_audio_enco
 			ret = iso_stream_send(enc_audio.data, data_size_pr_stream, bap_streams[i],
 					      &tx_info_arr[i], common_tx_sync_ts_us);
 		} else {
-			ret = iso_stream_send(&enc_audio.data[data_size_pr_stream * i],
-					      data_size_pr_stream, bap_streams[i], &tx_info_arr[i],
-					      common_tx_sync_ts_us);
+			ret = iso_stream_send(
```

Comment:
The value of the mapping mask should be checked against the number of streams.

Reply:
No, it is purely what channel to send, so it should be checked against the size of encoded data coming in:
```
if (channel_mask[i] >= enc_audio.num_ch) {
      LOG_ERR("DANGER DANGER");
}
```

I see now that I also have to change the check that `(enc_audio.num_ch == streams_to_tx)` since we could have stereo encoded data, but want to send to three streams. In that case it would try to divide the audio in three, meaning none of the streams get a full frame.